pkg/authorization/api/types.go

```diff
@@ -298,6 +302,10 @@ type LocalSubjectAccessReview struct {
 	User string
 	// Groups is optional.  Groups is the list of groups to which the User belongs.
 	Groups sets.String
+	// Scopes to use for the evaluation.  Empty means, assume no scopes.
```

Comment:
maybe I'm just tired, but this took me a few times to parse. maybe something like `Empty means "use the unscoped (full) permissions of the user/groups".` would be clearer?